backend/controller/login.go

```diff
@@ -78,59 +59,80 @@ func LoginResponse(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if user.IsAdmin {
-		// Get the session data stored from the function above
-		sessionData := models.SessionData{
-			UserId: user.WebAuthnID(),
-		}
-		result := db.DB.First(&sessionData)
-		if result.Error == nil {
-			result = db.DB.Delete(&sessionData)
-		}
-		if result.Error != nil {
-			helpers.DBErrorHandling(result.Error, w)
-			return
-		}
-
-		session := webauthn2.SessionData{
-			Challenge:            sessionData.Challenge,
-			UserID:               sessionData.UserId,
-			AllowedCredentialIDs: sessionData.AllowedCredentialIds,
-			Expires:              sessionData.Expires,
-			UserVerification:     sessionData.UserVerification,
-			Extensions:           sessionData.Extensions,
-		}
-
-		credential, err := config.WebAuthn.FinishLogin(user, session, r)
-		if err != nil {
-			http.Error(w, "Could not finish log in", http.StatusInternalServerError)
-			return
-		}
-
-		err = SaveDBCredential(w, credential)
-		if err != nil {
-			return
-		}
-
-		token, err := createToken(user)
-
-		if err != nil {
-			http.Error(w, "Could not generate validation token", http.StatusInternalServerError)
-			return
-		}
-
-		payload := struct {
-			Token string `json:"token"`
-		}{
-			token,
-		}
-
-		errJson := json.NewEncoder(w).Encode(payload)
-		if errJson != nil {
-			http.Error(w, "Could not return results", http.StatusInternalServerError)
-			return
-		}
-	} else {
-		http.Error(w, "User not admin", http.StatusUnauthorized)
+	// Get the session data stored from the function above
+	sessionData := models.SessionData{
+		UserId: user.WebAuthnID(),
+	}
+	result := db.DB.First(&sessionData)
```

Comment:
did you verify sql for this line, if you see other places we are querying by webauthnID we use take example in RegisterResponse(): 	result := db.DB.Where(models.SessionData{UserId: user.WebAuthnID()}).Take(&sessionData)

Reply:
This is getting refactored in my PR @cameronkinsella you can just copy / paste these lines:
https://github.com/lynx-locks/lynx-web/blob/7392f230d61794884b003bc0e707c03e9ad4ab23/backend/controller/login.go#L53-L89

Reply:
This was not modified. This diff is a bit misleading because this section is no longer under the `if user.IsAdmin` branch, since this login works for both users and admin.

Reply:
i see, that is probably still a bug tho considering it was a bug on register

Reply:
It is modified @cameronkinsella

Reply:
I can add it if that's easier